docs/src/guides/run-gisaid-ingest-on-terra.rst

```diff
@@ -0,0 +1,90 @@
+*******************************************
+Ingest SARS-CoV-2 data from GISAID on Terra
+*******************************************
+
+We have provided two pipelines for importing data into Terra:
+
+* GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.
+* **GISAID Ingest - pull a private dataset if a user has their own API endpoint, username, and password.**
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagrammed below:
+
+.. image:: ../images/terra-ingest.png
+
+This guide describes the **GISAID Ingest**. 
+
+Import the GISAID ingest wdl workflow from Dockstore
+=====================================================
+
+1. `Set up a Terra account <https://terra.bio/>`_.
+2. Navigate to the following in Dockstore:
+    - `nextstrain/ncov/gisaid_ingest`_ for private (GISAID) data. Requires access to the GISAID API endpoint.
```

Comment:
Same suggestion above applies here.

---

Also,

> Requires access to the GISAID API endpoint.

Suggestion: Remove it from here (it is not Dockstore-specific) and instead have a `.. warning::` at the top of this page with a brief description on what the endpoint is used for, how to check if you have one, and how to obtain access (if that's clear?).

Reply:
Done with 5d4fc05554773023651c7d1849818c593b470722